sig-usability/charter.md

```diff
@@ -0,0 +1,54 @@
+# SIG Usability Charter
+
+This charter adheres to the conventions described in the [Kubernetes Charter README] and uses
+the Roles and Organization Management outlined in [sig-governance].
+
+## Scope
+
+The scope of SIG usability is the core end-user usability of the Kubernetes project. Thiis covers
+topics like user experience and accessibility. The goals of the SIG are to ensure that the Kubernetes
+project is maximaly usable by users of a variety of different backgrounds and abilities. Examples
+of efforts include internationalization and accessibility of documentation.
+
+### In scope
+
+#### Code, Binaries and Services
+
+- usability of end-user facing experiences (error messages, end-to-end tasks, etc)
+- accessibility of Kubernetes community artifacts, examples include:
+   - internationalization of documentation in binaries and documentation
+   - color choices for people with color blind-ness
+   - ensuring compatibility with screen reader technology
+- user interface design for core components with user interfaces.
+- translations - https://github.com/kubernetes/kubernetes/tree/master/translations
+- internationalization - https://github.com/kubernetes/kubernetes/tree/master/pkg/kubectl/util/i18n
```

Comment:
The list of code, binaries, and services highlights areas we are often concerned with. While I wouldn't expect SIG Usability to own any of these, I would expect them to own UX profiles or personas (as the material gets mature enough to do personas) and some other usability meta things. How should we capture that?

Reply:
Suggestion:
### In scope

#### Repos
Suggest we create a separate user research repo for documentation of:
- Common end user profiles (aka personas)
- Ongoing list of research studies and findings
- Ongoing list of known needs for user research (ex: top stackoverflow questions)
- Curated research on usability best practices (ex: HCI research literature on interface usability)
- Templates for anybody to be able to conduct user research interviews

Suggest we create a separate accessibility repo for documentation of:
the bullet points already mentioned

Reply:
done.